### PR TITLE
Add new store model, adjust table selected row

### PR DESCRIPTION
**Pull request by macrog**
Id has been removed from the visit table, we will be using from now mongodb `_id`.
All should work as before!!

image

### CODE REVIEW

File: src/app/models/visit.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export interface Visit {
     patientId: string;
     note: string;
-    points: number[];
+    points: string[];
```
Comment (Johngtka) [inline src/app/models/visit.ts:4]: why string array, may be because the  _id field in backend is string and objects in all backend is array

**Review by Johngtka — COMMENTED** [review]
i thing all is good working

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -12,7 +12,10 @@ import { UserInputDialogComponent } from '../user-input-dialog/user-input-dial.c
     styleUrls: ['./home.component.css'],
 })
 export class HomeComponent {
-    constructor(private dialog: MatDialog, private route: Router) {}
+    constructor(
+        private dialog: MatDialog,
+        private route: Router,
+    ) {}
```
Comment (Johngtka) [inline src/app/home/home.component.ts:18]: did you use the npm format?
when i use it i will get this code `constructor(private dialog: MatDialog, private route: Router) {}` in one line
and more changes even in index.html like that `<!doctype html>`

**Review by Johngtka — COMMENTED** [review]
little clarifications

File: src/app/models/visit.ts (hunk 1 above)
Comment (macrog) [reply src/app/models/visit.ts:4]: this will change later on as we going to be storing a different things for visit

**Pull request merged**